README.md

```
@@ -86,43 +86,43 @@ gibidify supports a YAML configuration file. Place it at:
 
 ```yaml
-fileSizeLimit: 5242880  # 5 MB
+fileSizeLimit: 5242880	# 5 MB
 ignoreDirectories:
-  - vendor
-  - node_modules
-  - .git
-  - dist
-  - build
-  - target
+	- vendor
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)